docs/configuration/vrf/index.rst

```diff
@@ -444,6 +444,13 @@ address-family.
    derived and should not be specified explicitly for either the source or
    destination VRF’s.
 
+.. cfgcmd:: set [vrf name <name>] protocols bgp interface
```

Comment:
would please erase the `[ ]`, i know it is a correct syntax but this is the vrf section a the coverage progress will filter the strings between `[ ]`

Reply:
Will be fixed by me